docs/topics/spider-middleware.rst

```diff
@@ -357,8 +353,6 @@ Whether to enable referer middleware.
 REFERRER_POLICY
 ^^^^^^^^^^^^^^^
 
-.. versionadded:: 1.4
```

Comment:
What's the policy of keeping old versionadded directives?

Reply:
I was going with major versions, but it should probably be the same as for deprecation removals.